src/components/dashboard/sidebar/OfferSummaryCard.jsx

```diff
@@ -4,7 +4,7 @@ import PropTypes from 'prop-types';
 import SidebarCard from './SidebarCard';
 import { OFFER_SUMMARY_TITLE, OFFER_SUMMARY_NOTICE } from './data/constants';
 
-const OfferSummaryCard = ({ offersCount, className }) => {
+const OfferSummaryCard = ({ offersCount, className, children }) => {
```

Comment:
Why do we pass and display children if we don't use them when we reference `OfferSummaryCard` ? 
The same question stands for `SubscriptionSummaryCard`.

Reply:
I was using this originally, will remove.